Libraries/Components/Navigation/NavigatorIOS.ios.js

```diff
@@ -25,6 +25,8 @@ var invariant = require('fbjs/lib/invariant');
 var logError = require('logError');
 var requireNativeComponent = require('requireNativeComponent');
 
+var keyMirror = require('fbjs/lib/keyMirror');
```

Comment:
The rest of the code is using `var`, so I’d be tempted to leave this to a later code clean up.